src/js/tabs/gps.js

```diff
@@ -253,36 +257,38 @@ gps.initialize = async function (callback) {
 
                     if (FC.GPS_DATA.chn[i] >= 7) {
                         rowContent += '<td>-</td>';
-                        rowContent += `<td><progress value="${0}" max="99"></progress></td>`;
+                        rowContent += `<td><meter value="${0}" max="55"></meter></td>`;
                         rowContent += `<td> </td>`;
                     } else {
                         rowContent += `<td>${FC.GPS_DATA.svid[i]}</td>`;
-                        rowContent += `<td><progress value="${FC.GPS_DATA.cno[i]}" max="99"></progress></td>`;
+                        rowContent += `<td><meter value="${FC.GPS_DATA.cno[i]}" max="55"></meter></td>`;
 
                         let quality = i18n.getMessage(qualityArray[FC.GPS_DATA.quality[i] & 0x7]);
                         let used = i18n.getMessage(usedArray[(FC.GPS_DATA.quality[i] & 0x8) >> 3]);
-                        let healthy = i18n.getMessage(healthyArray[(FC.GPS_DATA.quality[i] & 0x30) >> 4]);
+                        let usedColor = '';
 
                         // Add color to the text
-                        if (quality.startsWith('fully locked')) {
+                        // 2nd column: no signal = red, unusable = red, searching = red, locked = yellow and fully locked = green
+                        if (quality.startsWith(i18n.getMessage('gnssQualityFullyLocked'))) {
+                            usedColor = 'locked';
                             quality = `<span class="colorToggle ready">${quality}</span>`;
                         } else {
-                            quality = `<span class="colorToggle">${quality}</span>`;
-                        }
-
-                        if (used.startsWith('used')) {
-                            used = `<span class="colorToggle ready">${used}</span>`;
-                        } else {
-                            used = `<span class="colorToggle">${used}</span>`;
+                            if (quality.startsWith(i18n.getMessage('gnssQualityLocked'))) {
```

Comment:
Fix your code by using else if if the nested if is the only statement in the else block.

Reply:
on it way :-)